cypress/integration/user.test.js

```diff
@@ -92,15 +92,14 @@ describe('User list and items test', () => {
       .click()
       .wait(2000);
 
-    cy.get('#status').should('have.text', 'Активний');
+    cy.get('#status').should('have.text', 'Активний(-a)');
   });
   it('Data from the server should be equal with the incoming data', () => {
     cy.visit(`users/${id}`);
-    cy.get('#firstName').should('have.text', firstName);
-    cy.get('#lastName').should('have.text', lastName);
+    cy.get('#name').should('have.text', `${firstName } ${ lastName}`);
```

Comment:
Please, don't use and `id` use `data-cy` attribute on components and then query it from cypress tests

Reply:
fixed